corehq/pillows/synclog.py

```diff
@@ -0,0 +1,139 @@
+from corehq.apps.receiverwrapper.util import get_version_and_app_from_build_id
+from corehq.apps.users.models import LastSync, CouchUser, CommCareUser, WebUser
+from corehq.pillows.utils import update_latest_builds, filter_by_app
+from corehq.util.doc_processor.interface import BaseDocProcessor, DocumentProcessorController
+from corehq.util.doc_processor.couch import CouchDocumentProvider
+
+from dimagi.utils.parsing import string_to_utc_datetime
+
+from pillowtop.pillow.interface import ConstructedPillow
+from pillowtop.processors.interface import PillowProcessor
+from pillowtop.feed.couch import CouchChangeFeed
+from pillowtop.feed.interface import Change
+from pillowtop.checkpoints.manager import PillowCheckpoint, PillowCheckpointEventHandler
+from pillowtop.reindexer.reindexer import Reindexer
+
+from casexml.apps.phone.models import SyncLog
+from casexml.apps.phone.dbaccessors.sync_logs_by_user import get_synclogs_for_user
+
+
+def get_synclog_pillow(pillow_id='SynclogPillow'):
```

Comment:
I think these should have more specific names like `UpdateUserSyncHistoryPillow` or something.